### PR TITLE
[Google Drive] change display annotation

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
### Fixes 
https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/issues/8
https://github.com/wso2-enterprise/choreo/issues/4421

image

### CODE REVIEW

File: drive/client.bal [hunk 1]
```diff
@@ -58,7 +58,7 @@ public client class Client {
     # + fileId - ID of the file to retreive
     # + return - If successful, returns `string`. Else returns `error`
     @display {label: "Download file"}
-    remote isolated function downloadFile(@display {label: "File id to delete"} string fileId) 
+    remote isolated function downloadFile(@display {label: "File id to download"} string fileId)
```
Comment (abeykoon) [inline drive/client.bal:61]: We know the operation is `Download File`. Why do we need `to download` part at the end of the parameter?

Comment (SkNuwanTissera) [reply drive/client.bal:61]: Yeah. That makes sense. I will update the PR as {label: "File id"}

Comment (SkNuwanTissera) [reply drive/client.bal]: Updated.. Thanks

**Comment by SkNuwanTissera** [comment]
Refer to new PR's
https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/pull/17 - To master 
https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/pull/18 - To slalpha5

**Pull request closed without merging**